plugins/obs-filters/cmake/speexdsp.cmake

```diff
@@ -6,8 +6,7 @@ if(ENABLE_SPEEXDSP)
   target_sources(obs-filters PRIVATE noise-suppress-filter.c)
   target_link_libraries(obs-filters PRIVATE SpeexDSP::Libspeexdsp)
   target_compile_definitions(obs-filters PRIVATE LIBSPEEXDSP_ENABLED)
-  target_link_options(obs-filters PRIVATE $<$<PLATFORM_ID:Windows>:/LTCG> $<$<PLATFORM_ID:Windows>:/IGNORE:4098>
-                      $<$<PLATFORM_ID:Windows>:/IGNORE:4099>)
+  target_link_options(obs-filters PRIVATE $<$<PLATFORM_ID:Windows>:/NODEFAULTLIB:MSVCRT>)
```

Comment:
This fails building right now locally and on CI, if I replace it with `msvcrtd.lib` as in `win-capture` it works: https://github.com/obsproject/obs-studio/blob/3e6797ca5bfe445381d721e407163b70b8de6d1e/plugins/win-capture/CMakeLists.txt#L80

Reply:
That kinda makes sense, I'm just weirded out that it didn't break in debug config for me locally.